plugins/leemons-plugin-tasks/frontend/src/components/TaskSetupPage/components/Submissions/components/File/File.js

```diff
@@ -36,7 +36,10 @@ export default function File({ labels }) {
             required
             error={errors.data?.extensions}
             onChange={(extensions) => {
-              const validExtensions = extensions.reduce((values, extension) => {
+              const cleanedExtensions = extensions.map((extension) =>
+                extension.replace(/[ .]/g, '')
```

Comment:
I would use `extension.trim()` as `.pdf` is accepted and the problem is with ` .pdf`
And I don't know if "double extensions" are official for mime: `.test.js`